python/xnvme-core/bin/xpy_libconf

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 import sys
 
-import xnvme.ctypes_mapping as xnvme
+import xnvme.ctypes_mapping.api as xnvme
```

Comment:
I think we should get rid of the `.api`. It can easily be done through a star-import in `__init__.py` or use the code generator for it.

Reply:
Yeah, I agree it would be nicer. Hmm, I did a quick star-import but it seems for some reason to have wider implications...

Reply:
Thanks for the offline input, will squash in the star-import.

Reply:
done.